src/TemplatePackExtension/source.extension.vsixmanifest

```diff
@@ -21,6 +23,6 @@
   </Assets>
   <Prerequisites>
     <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
-    <Prerequisite Id="Microsoft.VisualStudio.Component.Web" Version="[15.0.26427,16.0)" DisplayName="ASP.NET and web development tools" />
```

Comment:
This is by design. It is required to have VS2017 update 3 installed for the extension to work.